lib/VisualGenerator.js

```diff
@@ -391,14 +392,16 @@ class VisualGenerator {
         });
         if (regexNumber.test(name)) {
             return `The visual name can't begin with a number digit`;
-
-        } else if (regexES3ReservedWord.test(valueAsUnescapedString)) {
+        } else
+        if (!regexpWrongSymbols.test(name)) {
```

Comment:
`if` should be on the same line with `else`.
Please consider this for all of files.